lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -157,7 +209,8 @@ shutdown_unknown:
   default_value: true
 
 shutdown_vlan:
-  default_value: true
+  nxapi:
+    default_value: true
```

Comment:
How does node_utils handle a missing YAML entry for `grpc`? Does it just ignore the property?